apps/client/src/app/qr/page.tsx

```diff
@@ -0,0 +1,18 @@
+'use-client'
+
+import QRCode from "react-qr-code";
+
+export default function Id() {
+    const UserId = '12345';
```

Comment:
According to JS naming conventions, variables should be named via camelCase (you can check here - https://www.w3schools.com/js/js_conventions.asp)

Reply:
Done.